mw-webapp/src/logic/waysTable/MentoringWaysTable.tsx

```diff
@@ -111,51 +108,37 @@ export const MentoringWaysTable = (props: MentoringWaysTableProps) => {
     cell: ({row}) => {
       return (
         <VerticalContainer>
-          {row.original.mentors.map((mentor) => (
-            <div key={uuidv4().concat(mentor.uuid)}>
-              <Link
-                key={mentor.uuid}
-                path={pages.user.getPath({uuid: mentor.uuid})}
-                value={mentor.name}
-              />
-              {props.uuid === mentor.uuid && (
-                <>
+          {row.original.mentors.map((mentor) => {
+            return (
+              <div key={uuidv4().concat(mentor.uuid)}>
+                <Link
+                  key={mentor.uuid}
+                  path={pages.user.getPath({uuid: mentor.uuid})}
+                  value={mentor.name}
+                />
+                {props.uuid === mentor.uuid && (
                   <TrashIcon
                     className={styles.icon}
-                    onClick={() =>
-                      setIsStopMentoringModalOpen(true)
-                    }
-                  />
-                  <Modal
-                    content={
-                      <>
-                        <p>
-                          Are you sure you want to stop Mentoring this Way?
-                        </p>
-                        <Button
-                          value="Confirm"
-                          onClick={() => {
-                            stopMentoring(props.uuid, row.original);
-                            setIsStopMentoringModalOpen(false);
-                          }}
-                        />
-                        <Button
-                          value="Cancel"
-                          onClick={() =>
-                            setIsStopMentoringModalOpen(false)
-                          }
-                        />
-                      </>
-                    }
-                    isOpen={isStopMentoringModalOpen}
-                    handleClose={() =>
-                      setIsStopMentoringModalOpen(false)
-                    }
+
+                    onClick={() => {
+
+                      /**
+                       * TODO: use modal instead of confirm task #305
```

Comment:
Put comment in the right place, remove multiline commet

Reply:
Placed next to confirm in one line